GeorgianMemenator/Scenes/CatTagsList/CatTagsListView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  CatTagsListView.swift
+//  GeorgianMemenator
+//
+//  Created by Giorgi Makadze on 09.01.2023.
+//  Copyright © 2023 Shalva Avanashvili. All rights reserved.
+//
+
+import SwiftUI
+import Foundation
+
+struct CatTagsListView: View {
+    @ObservedObject var viewModel: CatTagsListViewModel
+    
+    var body: some View {
+        EmptyView()
+    }
+}
+
+struct CatTagsListView_Previews: PreviewProvider {
+    static var previews: some View {
+        MainView(
```

Comment:
`CatTagsListView_Previews` should construct `CatTagsListView` for preview

Reply:
I dont understand this instructions. Please will explain me more concretely.

Reply:
Check what does Reddit preview shows.

`CatTagsListView_Previews` is used to show the cat tags list view preview screen on the right. Instead of showing `CatTagsListView` your preview shows `MainView`, so:

```swift
struct CatTagsListView_Previews: PreviewProvider {
    static var previews: some View {
        // instantiate CatTagsListView not MainView
    }
```

Reply:
struct Reddit_Previews: PreviewProvider {
    static var previews: some View {
        MainView(
            viewModel: MainViewModel()
        )
    }
This is a Reddit preview shows.

Reply:
Ah I see, its a mistake on Reddit then, please instantiate the cat tags list view in the preview. See how @qetiavanashvili  did it [here](https://github.com/ashalva/ios.memenator/blob/18bd907cee4ed58cfd0fbc0cec47c7c5e0ec92f8/GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift#L23)

Reply:
It's okay, I fixed it.